DOKS-wordpress/README.md

```diff
@@ -150,6 +151,143 @@ Finally, you need to setup the trusted sources between your MySQL database and y
 
 Please visit [How to Secure MySQL Managed Database Clusters](https://docs.digitalocean.com/products/databases/mysql/how-to/secure/) for more details.
 
+## Installing and configuring the OpenEBS Dynamic NFS Provisioner
+
+A new [DigitalOcean Block Storage](https://docs.digitalocean.com/products/volumes/) volume is provisioned each time you use a [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) as part of your Kubernetes stateful application. The [StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) resource tells Kubernetes about the underlying storage type available. DigitalOcean is using [do-block-storage](https://github.com/digitalocean/csi-digitalocean) by default.
+
+Below command lists the available storage classes for your Kubernetes cluster:
+
+```console
+kubectl get sc
+```
+
+The output looks similar to:
+
+```text
+NAME                         PROVISIONER                 RECLAIMPOLICY   VOLUMEBINDINGMODE   ALLOWVOLUMEEXPANSION   AGE
+do-block-storage (default)   dobs.csi.digitalocean.com   Delete          Immediate           true                   24h
+```
+
+DigitalOcean Block Storage Volumes are mounted as read-write by a single node (RWO). Additional nodes cannot mount the same volume. The data content of a PersistentVolume can not be accessed by multiple Pods simultaneously.
+
+Horizontal pod autoscaling (HPA) policies are used to scale the WordPress Pods in a dynamically StatefulSet.
+
+This is why WordPress requires a [volume](https://kubernetes.io/docs/concepts/storage/volumes/) that can be mounted as read-write by many nodes (RWX), in your case, WordPress replica Pods.
+
+NFS (Network File System) is a commonly used solution to provide RWX volumes on block storage. This server offers a PersistentVolumeClaim (PVC) in RWX mode so that multiple web applications can access the data in a shared fashion.
+
+OpenEBS Dynamic NFS Provisioner allows users to create an NFS PV that sets up a new Kernel NFS instance for each PV on top of the user's choice of backend storage.
+
+**Note:**
+
+Please visit [OpenEBS](https://openebs.io/) for more details.
+
+Next, you will install the OpenEBS Dynamic NFS Provisioner on your Kubernetes cluster using the [OpenEBS Helm Chart](https://github.com/openebs/dynamic-nfs-provisioner). For your purposes you will be installing and configuring only the dynamic nfs provisioner since that's what you need.
```

Comment:
This sentence begins a little bit odd for me:

> For your purposes you will be installing and configuring only the dynamic nfs provisioner since that's what you need.

I think it can be written as:

>You will be installing and configuring only the dynamic nfs provisioner, since Wordpress requires it.